.env.jenkins

```diff
@@ -16,4 +16,4 @@ PFS_ENCRYPTION_ENABLED=1
 PAIRING_ENABLED=1
 ERC20_CONTRACT_WARNINGS=1
 MAILSERVER_CONFIRMATIONS_ENABLED=0
-HARDWALLET_ENABLED=0
+HARDWALLET_ENABLED=1
```

Comment:
Do we still need it?

Reply:
@jeluard Yes, it's needed to enable this feature during QA. I will disable it back once testing is finished.